OmniKit/MessageTransport/MessageBlocks/PodInfoFaultEvent.swift

```diff
@@ -96,27 +97,28 @@ public struct PodInfoFaultEvent : PodInfo, Equatable {
 
 extension PodInfoFaultEvent: CustomDebugStringConvertible {
     public typealias RawValue = Data
+    
     public var debugDescription: String {
         return [
             "## PodInfoFaultEvent",
             "* rawHex: \(data.hexadecimalString)",
             "* podProgressStatus: \(podProgressStatus)",
             "* deliveryStatus: \(deliveryStatus.description)",
-            "* insulinNotDelivered: \(insulinNotDelivered) U",
-            "* podMessageCounter: \(podMessageCounter)",
-            "* unknownPageCode: \(unknownPageCode.hexadecimalString)",
+            "* insulinNotDelivered: (%.2f, \(insulinNotDelivered)) U",
+            "* podMessageCounter: 0x\(podMessageCounter)",
```

Comment:
The 0x prefix implies, hex, but this is rendering decimal

Reply:
kept decimal to render, only used it for unknownValue